parsl/monitoring/db_manager.py

```diff
@@ -236,6 +244,55 @@ class Block(Base):
             PrimaryKeyConstraint('run_id', 'block_id', 'executor_label', 'timestamp'),
         )
 
+    class Files(Base):
+        __tablename__ = FILES
+        file_name = Column('file_name', Text, index=True, nullable=False)
+        file_path = Column('file_path', Text, nullable=True)
+        file_id = Column('file_id', Text, index=True, nullable=False)
+        run_id = Column('run_id', Text, nullable=False)
+        task_id = Column('task_id', Integer, nullable=True)
+        try_id = Column('try_id', Integer, nullable=True)
+        timestamp = Column('timestamp', DateTime, nullable=True)
+        size = Column('size', BigInteger, nullable=True)
+        md5sum = Column('md5sum', Text, nullable=True)
+        sa.Index("files_task_run_id_idx", "task_run_id", "task_id", "try_id")
+        __table_args__ = (PrimaryKeyConstraint('file_id'),)
+
+    class Environment(Base):
+        __tablename__ = ENVIRONMENT
+        environment_id = Column('environment_id', Text, nullable=False)
+        run_id = Column('run_id', Text, nullable=False)
+        label = Column('label', Text, nullable=False)
+        address = Column('address', Text, nullable=True)
+        provider = Column('provider', Text, nullable=True)
+        launcher = Column('launcher', Text, nullable=True)
+        worker_init = Column('worker_init', Text, nullable=True)
+        __table_args__ = (PrimaryKeyConstraint('environment_id'),)
+
+    class InputFiles(Base):
```

Comment:
probably should be singular not plural: name the think that is described by a single row in the table (see how a `TASK` row describes a single task)